package.json

```diff
@@ -1,20 +1,21 @@
 {
   "name": "docker-ttrss-plugins",
   "version": "1.0.0",
+  "private": true,
   "repository": "git@github.com:HenryQW/docker-ttrss-plugins.git",
-  "author": "Henry <hi@henry.wang>",
   "license": "MIT",
-  "private": true,
+  "author": "Henry <hi@henry.wang>",
   "scripts": {
-    "dev": "vuepress dev docs",
-    "docs:build": "vuepress build docs",
-    "format": "prettier \"**/*.{js,json,md}\" --write",
+    "docker:build": "docker rm ttrss -f && docker-compose -f docker-compose.dev.yml up -d --build && docker logs ttrss -f",
     "docker:dev": "docker rm ttrss -f && docker-compose -f docker-compose.yml up -d && docker logs ttrss -f",
-    "docker:build": "docker rm ttrss -f && docker-compose -f docker-compose.dev.yml up -d --build && docker logs ttrss -f"
+    "docs:build": "NODE_OPTIONS=--openssl-legacy-provider vuepress build docs",
+    "docs:dev": "NODE_OPTIONS=--openssl-legacy-provider vuepress dev docs",
+    "format": "prettier \"**/*.{js,json,md}\" --write"
   },
   "devDependencies": {
     "@vuepress/plugin-back-to-top": "1.9.10",
     "@vuepress/plugin-google-analytics": "1.9.10",
     "vuepress": "1.9.10"
-  }
+  },
+  "packageManager": "pnpm@8.15.7+sha256.50783dd0fa303852de2dd1557cd4b9f07cb5b018154a6e76d0f40635d6cee019"
```

Comment:
pinned to v8 until https://github.com/dependabot/dependabot-core/issues/9522 is fixed